testing/resources/types_or_repo/.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+-   id: python-cython-files
+    name: Python and Cython files
+    entry: bin/hook.sh
+    language: script
+    types: [file]
+    types_or: [python, cython]
```

Comment:
these are usually easier to demo with a local repo

I'd probably not write a full integration test for this feature as well but it looks like there maybe aren't great examples using the matcher

Reply:
Thanks for your feedback, but I'm not sure I understand - are there any existing tests I should look at as examples? If not, I'll try to figure this out, no worries